README.md

```diff
@@ -21,3 +21,18 @@ This will install bolt to `/opt/puppetlabs/bin/bolt`, so to use bolt add `/opt/p
 ```
 export PATH=$PATH:/opt/puppetlabs/bin
 ```
+
+### PDK
+
+To install [PDK](https://github.com/puppetlabs/pdk) with brew:
+
+```
+brew cask install puppetlabs/puppet/pdk
+```
+
+This will install PDK into `/opt/puppetlabs/pdk` and add an entry into `/etc/paths.d` to add PDK to your
```

Comment:
`/opt/puppetlabs/pdk/bin` ? On OSX 12 at least I had to manually add to my `PATH`. Do I need to restart the shell or reload anything to load `/etc/paths.d`?

Reply:
Derp, I just saw the "may have to relaunch your session" :P LGTM!